pkg/scheduler/factory_test.go

```diff
@@ -539,32 +540,30 @@ func newConfigFactoryWithFrameworkRegistry(
 	client clientset.Interface, hardPodAffinitySymmetricWeight int32, stopCh <-chan struct{},
 	registry framework.Registry, pluginConfigProducerRegistry *frameworkplugins.ConfigProducerRegistry) *Configurator {
 	informerFactory := informers.NewSharedInformerFactory(client, 0)
-	return NewConfigFactory(&ConfigFactoryArgs{
-		Client:                         client,
-		InformerFactory:                informerFactory,
-		NodeInformer:                   informerFactory.Core().V1().Nodes(),
-		PodInformer:                    informerFactory.Core().V1().Pods(),
-		PvInformer:                     informerFactory.Core().V1().PersistentVolumes(),
-		PvcInformer:                    informerFactory.Core().V1().PersistentVolumeClaims(),
-		ReplicationControllerInformer:  informerFactory.Core().V1().ReplicationControllers(),
-		ReplicaSetInformer:             informerFactory.Apps().V1().ReplicaSets(),
-		StatefulSetInformer:            informerFactory.Apps().V1().StatefulSets(),
-		ServiceInformer:                informerFactory.Core().V1().Services(),
-		PdbInformer:                    informerFactory.Policy().V1beta1().PodDisruptionBudgets(),
-		StorageClassInformer:           informerFactory.Storage().V1().StorageClasses(),
-		CSINodeInformer:                informerFactory.Storage().V1().CSINodes(),
-		HardPodAffinitySymmetricWeight: hardPodAffinitySymmetricWeight,
-		DisablePreemption:              disablePodPreemption,
-		PercentageOfNodesToScore:       schedulerapi.DefaultPercentageOfNodesToScore,
-		BindTimeoutSeconds:             bindTimeoutSeconds,
-		PodInitialBackoffSeconds:       podInitialBackoffDurationSeconds,
-		PodMaxBackoffSeconds:           podMaxBackoffDurationSeconds,
-		StopCh:                         stopCh,
-		Registry:                       registry,
-		Plugins:                        nil,
-		PluginConfig:                   []schedulerapi.PluginConfig{},
-		PluginConfigProducerRegistry:   pluginConfigProducerRegistry,
-	})
+	snapshot := nodeinfosnapshot.NewEmptySnapshot()
+	return &Configurator{
```

Comment:
This is missing `enableNonPreempting`, but it's hard to see if there's anything else missing. Can you think of a way of avoiding this?

Reply:
Unit tests is supposed to capture this if this is a parameter that is actually tested. It seems that we don't quite care about the actual parameters in this test file.